src/configurePipeline/utility/pipelineHelper.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import * as vscode from 'vscode';
+import { PipelineTargets } from "../model/Common";
+
+export async function getSelectedPipeline(nodeUri: vscode.Uri[], dockerUri: vscode.Uri[]): Promise<string> {
+	let items: string[] = [];
+	if (nodeUri.length === 0) {
+		throw new Error("Failed to detect Node.js application");
+	}
+	else if (dockerUri.length > 0) {
+		items = [NodeOnContainers, NodeOnWindows];
+	}
+	else {
+		items = [NodeOnWindows, NodeOnContainers];
+	}
+	return vscode.window.showQuickPick(items, {
+		placeHolder: "Select Azure pipelines template .."
+	});
+}
+
+export async function analyzeRepo() {
```

Comment:
This can take path as a parameter as we have option for current workspace or user can select some other git folder